src/data/moves_info.h

```diff
@@ -122,15 +122,15 @@ const struct MoveInfo gMovesInfo[MOVES_COUNT_DYNAMAX] =
         .name = COMPOUND_STRING("-"),
         .description = COMPOUND_STRING(""),
         .effect = EFFECT_HIT,
-        .power = 0,
         .type = TYPE_NORMAL,
+        .category = DAMAGE_CATEGORY_PHYSICAL,
+        .power = 0,
         .accuracy = 0,
-        .pp = 0,
         .target = MOVE_TARGET_SELECTED,
+        .pp = 0,
         .priority = 0,
-        .category = DAMAGE_CATEGORY_PHYSICAL,
-        .metronomeBanned = TRUE,
         .mirrorMoveBanned = TRUE,
```

Comment:
Ok, I need to ask this.

*Why?*

Why change the stablished order? I get grouping banned flags together, but this will create so many conflicts across the board that I don't get why you chose to do this.

Reply:
To put everything into a more sensible order and make better use of space. You approved this when I asked about it ages ago?

Reply:
![Screenshot_20240213-081302_Discord (1)](https://github.com/rh-hideout/pokeemerald-expansion/assets/22523466/f07db5ec-18c8-47c7-b158-e96ff8614f84)

Tbf, I think you were more interested in the flag rearrangement, but all of the above was in the proposal, for which you said this ^

Reply:
Besides, we've cause conflicts before by renaming `split` (and then also the defines - twice), adding contest data and names, and adding `additionalEffects`. This would probably add more conflicts than any of those but it would be worth it to establish, IMO, now that all move data fields have been added to the struct, a final, clean, sensible order.

Reply:
> Tbf, I think you were more interested in the flag rearrangement, but all of the above was in the proposal, for which you said this ^

From what I remember, you only showed the struct rearrangement, not that you were going to change the field order in the data as well.

Reply:
> Besides, we've cause conflicts before by renaming `split` (and then also the defines - twice), adding contest data and names, and adding `additionalEffects`. This would probably add more conflicts than any of those but it would be worth it to establish, IMO, now that all move data fields have been added to the struct, a final, clean, sensible order.

The current order is fine. Vanilla fields on top, new fields at the bottom.

Reply:
Oh! I thought that having moves info and the struct be in synch was preferred. Has that not been the case?

Reply:
In fact, I seem to recall with the species form refactor how you didn't like that the order of the fields had been lost (and you were satisfied when I then send that I would rearrange them again before finishing).

Reply:
> The current order is fine to me. Vanilla fields on top, new fields at the bottom.

They aren't really, tbh. The zMove field is way up there, and the flags are in kind of a random order as well (not that I'm willing to touch those but still).

Reply:
> In fact, I seem to recall with the species form refactor how you didn't like that the order of the fields had been lost (and you were satisfied when I then said that I would rearrange them again before finishing).

I don't recall this much. In the end, the struct was arranged to minimize space used but then the fields were organized such that Dex data and graphics data were all next to each other, despite not that being how it's setup in the struct.

Reply:
@AsparagusEduardo This is what you said back then and it's why I spent my time reordering moves_info to match the struct:
![Screenshot_20240213-090257_Edge (1)](https://github.com/rh-hideout/pokeemerald-expansion/assets/22523466/e4955bf8-b520-466a-9cc1-f860c08f877e)

Reply:
Because this was an automated thing you did that moved fields that were previously in species macros to the top of the array. All mon that didn't have macros kept their previous order, so it was inconsistent.

Reply:
In fact, the macros I made were curated to keep the field order that I had established for all non-macro Pokémon, so base stats on top and so on.

Reply:
I need to gather my thoughts. I've been outside of the house the whole day and I can't focus.

Reply:
Reverted the change since we can do it later anyway